app2/src/main/java/com/example/recipebox/MainActivity.kt

```diff
@@ -4,9 +4,7 @@ import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
+import androidx.compose.material.*
```

Comment:
I'd leave this as it were.  Wildcard imports (*) are considered bad practice for a couple of reasons:
1) Less documentation on what classes are being imported.
2) If you have two wildcard imports, and they have a class with the same name, there will be a collision.  Example: java.util.* and java.sql.* both have a class named Date.  Different classes, same name.  So, if you're using wildcards, which Date class should be used?